client/src/components/Card/Card.stories.js

```diff
@@ -4,13 +4,18 @@ import Card from "./Card";
 import image from "../../../../assets/images/card_small3.png"; //here for example
 
 storiesOf("Card", module)
-  .add("Info Card, Purple", () => <Card color="purple" size="large" />)
-  .add("Info Card, Purple, Small", () => <Card color="purple" size="small" />)
-  .add("Info Card, Pink", () => <Card color="pink" size="large" />)
-  .add("Info Card, Pink, Small", () => <Card color="pink" size="small" />)
-  .add("Info Card, Blank", () => <Card size="large" />)
-  .add("Info Card, Blank, Small", () => <Card size="small" />)
-  .add("Info Card, Picture BG, Large", () => <Card picture={image} alt="whoops" size="medium" />)
-  .add("Info Card, Picture Hawaii, Large", () => (
+  .add("Card, Purple", () => <Card color="purple" size="large" />)
+  .add("Card, Purple, Small", () => <Card color="purple" size="small" />)
+  .add("Card, Pink", () => <Card color="pink" size="large" />)
+  .add("Card, Pink, Small", () => <Card color="pink" size="small" />)
+  .add("Card, Blank", () => <Card size="large" />)
+  .add("Card, Blank, Small", () => <Card size="small" />)
+  .add("Card, Picture BG, Large", () => <Card picture={image} alt="whoops" size="medium" />)
+  .add("Card, Picture Hawaii, Large", () => (
     <Card picture={"https://wallpaperaccess.com/full/144067.jpg"} alt="whoops" size="medium" />
+  ))
+  .add("Card, Discover New Groups", () => (
+    <Card discover={true} size="medium">
```

Comment:
Could we call the prop something more semantic and less `contextual`? i.e. `border`, `outline` or something like that?